src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -671,7 +671,26 @@ public GHAuthorization createToken(Collection<String> scope, String note, String
 
         return requester.method("POST").to("/authorizations", GHAuthorization.class).wrap(this);
     }
-
+    /**
+     * Creates a new authorization using an OTP.
+     * 
+     * Start by running createToken, if exception is thrown, prompt for OTP from user
```

Comment:
This is what happens on the client side, right?

Reply:
This is the stateful use explanation. Since this is unlike any other operation in that it is stateful (request token -> rejection prompts 2f4 OTP generation server side -> user receives OTP -> user calls this function with the time sensitive OTP and this time it is not rejected and the token is returned.

Reply:
This raises a sticky question, how do i write a test for a stateful interaction between 2 functions like this?